FiveCalls/FiveCalls/ScriptReplacements.swift

```diff
@@ -0,0 +1,61 @@
+//
+//  ScriptReplacements.swift
+//  FiveCalls
+//
+//  Created by Nick O'Neill on 12/12/23.
+//  Copyright © 2023 5calls. All rights reserved.
+//
+
+import Foundation
+
+struct ScriptReplacements {
```

Comment:
Touching this could be an opportunity to use the new Swift Regex system added in iOS 16. I might take this on as a follow up myself to get more familiar with it.

Reply:
That would be great, I had no idea there was a more modern way to regex!